vault/expiration.go

```diff
@@ -2295,16 +2304,19 @@ func (m *ExpirationManager) lookupLeasesByToken(ctx context.Context, te *logical
 
 // emitMetrics is invoked periodically to emit statistics
 func (m *ExpirationManager) emitMetrics() {
-	// All updates of this value are with the pendingLock held.
+	// All updates of these values are with the pendingLock held.
 	m.pendingLock.RLock()
-	num := m.leaseCount
+	allLeases := m.leaseCount
+	irrevocableLeases := m.irrevocableLeaseCount
 	m.pendingLock.RUnlock()
 
-	metrics.SetGauge([]string{"expire", "num_leases"}, float32(num))
+	metrics.SetGauge([]string{"expire", "num_leases"}, float32(allLeases))
+
+	metrics.SetGauge([]string{"expire", "num_irrevocable_leases"}, float32(irrevocableLeases))
```

Comment:
we should either change this to `irrevocable_leases` or update VLT-145 (in an Appendix -> Changes bullet point?) to say the metric is `num_irrevocable_leases`. I'm fine with either, but I like having the `num_` prefix since it's more in-line with what we are doing for all leases

Reply:
Updated the RFC with this metric name